ui/js/actions/content.js

```diff
@@ -505,3 +505,42 @@ export function doPublish(params) {
     });
   };
 }
+
+export function doAbandonClaim(claimId, name, txid, nout) {
+  return function(dispatch, getState) {
+    const state = getState();
+
+    dispatch({
+      type: types.ABANDON_CLAIM_STARTED,
+      data: {
+        claimId: claimId,
+      },
+    });
+
+    const errorCallback = error => {
+      dispatch(doOpenModal(modals.TRANSACTION_FAILED));
```

Comment:
Is it safe to show this modal for all failures?

Reply:
Since an abandoning txn is essentially a txn, so I figured that a `TRANSACTION_FAILED` modal would be apt. We could maybe go for a better error message altogether for a failed txn though. Or did you have something else that could be better in this case?